lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ConfigurationManager.cs

```diff
@@ -115,6 +119,17 @@ internal sealed class ConfigurationManager
             {
                 Exceptions.Throw(e, "Unable to deserialize map input pipeline data converter configuration", Logger);
             }
+
+            try
+            {
+                _resultHandlerConfiguration =
+                    configurationSerializer.FromString(resultHandlerConfiguration);
+                Logger.Log(Level.Info,"Serialized result handler is " + resultHandlerConfiguration);
+            }
+            catch (Exception e)
+            {
+                Exceptions.Throw(e, "Unable to deserialize map input pipeline data converter configuration", Logger);
```

Comment:
Me too :)....just using it to be consistent with the rest of code...let's have a discussion on dev list.